client/client_auth.go

```diff
@@ -93,6 +93,10 @@ func (u *AuthClient) Request(api ApiRequester, server string, dataSrc AuthDataMe
 
 	log.Debugf("got response: %v", rsp)
 
+	if rsp.StatusCode >= 400 && rsp.StatusCode < 600 {
```

Comment:
The ticket says "Include request_id details in standard Mender logging", but here I can see we are logging only 4XX and 5XX errors. Are you sure this was the intention of the ticket?

Reply:
as far as I can understand https://docs.mender.io/1.3/apis/device-apis/device-authentication#error only these codes are errors, and thus contain an error message.